doc/design/libsvm_format_column.md

```diff
@@ -0,0 +1,112 @@
+# Support LibSVM format on SQLFlow
+
+## Background
+
+SQLFlow extends SQL syntax to support End-to-End machine learning pipeline. 
```

Comment:
End-to-End => end-to-end.  Use grammarly.com pleaase.

Reply:
Done.